announcements/2021/week14.md

```diff
@@ -0,0 +1,16 @@
+# :zap: This week in provider-independent announcements (April 9, 2021) :zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, workload clusters, UI, and documentation, please check out [changes and releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed apps
+
+[Cert Manager v2.4.4](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.4.4/) renames the clusterissuer subchart and makes pods of deployments use read-only file systems, in order to...
```

Comment:
@QuentinBisson @glitchcrab @ubergesundheit , can you help clarify what the benefit of this release is? 😁  https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#244---2021-04-06

Reply:
What was the previous release announced? This is a very minor patch release which only changed internal stuff.

Reply:
since you confirmed, this wont affect customers and not interesting to them, deleted it!